### ISSUE TITLE
[DEV]Using rnetboot on OpenPower Systems does not give a good help message

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#873]
To network boot OpenPower LE systems, we should be using ``rsetboot <node> net``, 

However, if the user tries to boot from network using ``rnetboot`` , the error message that prints out is not informative enough to help the user do the right thing..

```
# rnetboot c910f05c31
c910f05c31: Error: Unable to identify plugin for this command, check relevant tables: nodehm.mgt

# tabdump nodehm | egrep "#|c31"
#node,power,mgt,cons,termserver,termport,conserver,serialport,serialspeed,serialflow,getmac,cmdmapping,consoleondemand,comments,disable
"c910f05c31",,"ipmi","ipmi",,,,,,,,,,,
```

In the case above, nodehm.mgmt is set, but really should be using ``rsetboot``. 

To follow up, probably need a better way for users to know which command to run depending on which architecture/hardware being used..??

**Comment by zet809** [comment]
@whowutwut Yes, currently, to set node boot from network, we have 3 commands
1. for ipmi based node, rsetboot <node> net
2. for fsp managed LPAR, the command rbootseq <node> net can be use
3, for HMC managed LPAR, shall use rnetboot <node>
I think we can have the persistent rnetboot command to support mgt=fsp and mgt=ipmi which means if mgt=fsp, it call "rbootseq net" internally, and for mgt=ipmi, it call "rsetboot net" internally.

**Comment by daniceexi** [comment]
Maybe an easier way to fix this is to change the error message to have an additional message like: 

~~~~
Error: Unable to identify plugin for this command, check relevant tables: nodehm.mgt. You may use incorrect command. Please check the map page of the command to make sure the command is suitable for your node.

~~~~

**Comment by daniceexi** [comment]
This error checking will be put in the **rinstall** command. Move to 2.12.1 to develop **rinstall** cmd.

**Comment by zet809** [comment]
Assign this defect to @penguhyang since he is working for rinstall command.

**Comment by penguhyang** [comment]
This bug has been fixed in #1157
